python/dask_cudf/dask_cudf/sorting.py

```diff
@@ -73,43 +74,41 @@ def _simple_shuffle(df, columns, npartitions, ignore_index=True):
             ignore_index,
             npartitions,
         )
-        for i in range(df.npartitions)
+        for i in df_nparitions
     }
 
+    _nparitions = set(range(npartitions))
```

Comment:
```suggestion
    _nparitions = tuple(range(npartitions))
```

Reply:
This should also be `_npartitions` I believe.

If this is guaranteed to be less than or equal to `df_npartitions` then we may be able to just slice `df_npartitions` to avoid materializing an extra tuple here.